sqlmesh/cli/example_project.py

```diff
@@ -23,30 +25,66 @@ def _gen_config(
     start: t.Optional[str],
     template: ProjectTemplate,
 ) -> str:
-    connection_settings = (
-        settings
-        or """      type: duckdb
+    if not settings:
+        connection_settings = """      type: duckdb
       database: db.db"""
-    )
+
+        doc_link = "# Visit https://sqlmesh.readthedocs.io/en/stable/integrations/engines{dialect_link} for more information on configuring the connection to your execution engine."
+
+        dialect_alias = "mssql" if dialect == "tsql" else dialect
```

Comment:
[Nit] Should this be `engine` instead? I don't think `mssql` refers to the _dialect_ in this case; that would be t-sql.